packages/flutter/lib/parse_server_sdk.dart

```diff
@@ -143,7 +143,7 @@ class CoreStoreSembastImp implements sdk.CoreStoreSembastImp {
   static sdk.CoreStoreSembastImp _sembastImp;
 
   static Future<sdk.CoreStore> getInstance(
-      {DatabaseFactory factory, String password}) async {
+      {DatabaseFactory factory, String password = 'flutter_sdk'}) async {
```

Comment:
I think this will avoid some frustration during development. 👍 
Do you think it would be a good idea to print a warning to the developer if the default password is used? (Only in debug mode using the assert-trick)

Reply:
I think it fits this PR.
Could you please add `?? 'flutter_sdk'` behind `masterKey` in this [line](https://github.com/parse-community/Parse-SDK-Flutter/blob/30ba45c86a17054b602c23ac3b718f2274db5b32/packages/flutter/lib/parse_server_sdk.dart#L89)?
(If you are busy, I can do it as well.)

Reply:
The parse_server_sdk package has a default password. When dividing the package into Dart / Flutter the password was not defined in the flutter and this causes an error in the initialization.
Yes. An alert might be interesting

Reply:
When I split the package, I did not realize, the value `null` would be 'forwarded'. As a result, the default password in the dart file is not used. My bad!

Something like this?
```dart
assert(() {
  if (password == 'flutter_sdk') {
    print('Warning: CoreStoreSembastImp uses the default password. Specify a custom one for increased security.');
  }
  return true;
}());
```
(Should be changed in packages/dart/lib/storage/core_store_sem_impl.dart)

Reply:
```dart

assert(() {
  if (password == 'flutter_sdk') {
   print('***********************************************************************************************************');
   print('Warning: CoreStoreSembastImp uses the default password. Specify a custom one for increased security.');
   print('***********************************************************************************************************');
  }
  return true;
}());

````

Reply:
👍 A little more visible.